packages/expo-video/src/VideoView.tsx

```diff
@@ -13,6 +13,11 @@ import NativeVideoModule from './NativeVideoModule';
 import NativeVideoView from './NativeVideoView';
 import { VideoPlayer, VideoSource, VideoViewProps } from './VideoView.types';
 
+/**
+ * Creates a `VideoPlayer`, which will be automatically cleaned up when the component is unmounted.
+ * @param source - A video source to initialize the player with
```

Comment:
```suggestion
 * @param source - A video source that is used to initialize the player.
```

Let's rephrase this to make it more clear and also add the missing period.